validphys2/src/validphys/closuretest/closure_results.py

```diff
@@ -394,3 +398,130 @@ def fit_underlying_pdfs_summary(fit, fitunderlyinglaw):
 def summarise_closure_underlying_pdfs(fits_underlying_pdfs_summary):
     """Collects the underlying pdfs for all fits and concatenates them into a single table"""
     return pd.concat(fits_underlying_pdfs_summary, axis=1)
+
+
+def covmat_diffs(data, inconsistent_datasets, sys_rescaling_factor):
+    """Computes the difference between the original covmat and a modified one given:
```

Comment:
Here please use the default way of writing the docs (you can find examples in the other functions)